setup_pubsub.sh

```diff
@@ -40,12 +40,30 @@ create_subscription() {
   fi
   echo -n "."
 }
+create_subscription_with_push_config() {
+  # Expects three arguments: project name, topic name, and subscription name
+  HTTP_RESPONSE=$(curl -s -w "HTTPSTATUS:%{http_code}" -X PUT "http://$PUBSUB_SETUP_HOST/v1/projects/$1/subscriptions/$3" -H "Content-Type: application/json" -d "{\"topic\": \"projects/$1/topics/$2\", \"pushConfig\":{\"pushEndpoint\":\"http://sdx-receipt-adapter:8080/projects/$1/topics/$2\" }}")
```

Comment:
given that this is a generic function we should pass in the push endpoint as a param